docs/sources/administration/service-accounts/add-service-account-token.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Add a token to a service account in Grafana'
+menuTitle: 'Add token for service account'
+aliases: [docs/sources/administration/service-accounts/add.md]
+description: 'This contains information about adding a token to a service account'
+weight: 60
+---
+
+# Add a service account token to a service account in Grafana
+
+You can add a token to service account in the service account tab of the organization.
```

Comment:
Remove this line because it is part of the steps below. In the intro, I would add a few sentences about what a token is.

Something like (and I've made this up, so adjust accordingly "A service account token a 15-character alphanumeric string that establishes communication between external systems and Grafana. Each service account has at least one token."